client/src/app/shared/models/constants.ts

```diff
@@ -127,6 +127,7 @@ export class Regex {
   public static readonly errorLog: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+)\ (\[Error|ERROR)/;
   public static readonly warningLog: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+)\ (\[Warning|WARNING)/;
   public static readonly log: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2})/;
+  public static readonly organizationName: RegExp = /^[^_.@~;{}\'+=,<>|/\\?*:&$*"#%\[\]][^@~;{}\'+=,<>|/\\?*:&$*"#%\[\]]*[^.@;{}\'+=,<>|/\\?*:&$*"#%\[\]]$/;
```

Comment:
Ensure that this is the same as the one used by AzDev portal or DevOps Project creation flow in Azure portal

Reply:
removed the validation as it doesn't affect the project name anymore or cause server failures